modules/op-modifying-catalog-refresh-interval-tekton-hub.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-catalog-refresh-interval-tekton-hub_{context}"]
+= Modifying the catalog refresh interval of {tekton-hub}
+
+[role="_abstract"]
+The default catalog refresh interval for {tekton-hub} is 30 minutes. Cluster administrators can modify the automatic catalog refresh interval by modifying the value of the `catalogRefreshInterval` field in the `TektonHub` CR.
+
+[discrete]
+.Procedure
+. Modify the value of the `catalogRefreshInterval` field in the `TektonHub` CR.
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonHub
+metadata:
+  name: hub
+spec:
+  targetNamespace: openshift-pipelines <1>
+  api:
+    catalogRefreshInterval: 30m <2>
+----
+<1> The namespace in which {tekton-hub} must be installed; default is `openshift-pipelines`.
```

Comment:
Are you saying {tekton-hub} must be installed in `openshift-pipelines`? If not, it might help to reword this.

Reply:
Maybe:
```suggestion
<1> The namespace where {tekton-hub} runs. The default value is `openshift-pipelines`.
```

Reply:
Yes, it is about installation. But I have reworded.